src/backend/distributed/executor/CitusExecutorLockMap.txt

```diff
@@ -0,0 +1,22 @@
+ReadOnly
```

Comment:
Well, we can consider not adding this and the PDF, as they might not be trivial to maintain.

Reply:
I'll either remove this file or move this to the code-comment. Any suggestion?